conda-forge.yml

```diff
@@ -4,4 +4,7 @@ conda_forge_output_validation: true
 github:
   branch_name: main
   tooling_branch_name: main
+provider:
+  linux_aarch64: default
```

Comment:
Instead of default, can you put `azure`

Reply:
For both of these. 

We are having issues with Travis right now.

Reply:
cc @lixun910 (I don't have commit rights here, I am a mere user)